Gutenberg/cocoapods_helpers.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+# Helpers and configurations for integrating Gutenberg in Jetpack and WordPress via CocoaPods.
+
+require_relative './version'
+
+DEFAULT_GUTENBERG_LOCATION = File.join(__dir__, '..', '..', 'gutenberg-mobile')
+
+# Note that the pods in this array might seem unused if you look for
+# `import` statements in this codebase. However, make sure to also check
+# whether they are used in the gutenberg-mobile and Gutenberg projects.
+#
+# See https://github.com/wordpress-mobile/gutenberg-mobile/issues/5025
+DEPENDENCIES = %w[
+  FBLazyVector
+  React
+  ReactCommon
+  RCTRequired
+  RCTTypeSafety
+  React-Core
+  React-CoreModules
+  React-RCTActionSheet
+  React-RCTAnimation
+  React-RCTBlob
+  React-RCTImage
+  React-RCTLinking
+  React-RCTNetwork
+  React-RCTSettings
+  React-RCTText
+  React-RCTVibration
+  React-callinvoker
+  React-cxxreact
+  React-jsinspector
+  React-jsi
+  React-jsiexecutor
+  React-logger
+  React-perflogger
+  React-runtimeexecutor
+  boost
+  Yoga
+  RCT-Folly
+  glog
+  react-native-safe-area
+  react-native-safe-area-context
+  react-native-video
+  react-native-webview
+  RNSVG
+  react-native-slider
+  BVLinearGradient
+  react-native-get-random-values
+  react-native-blur
+  RNScreens
+  RNReanimated
+  RNGestureHandler
+  RNCMaskedView
+  RNCClipboard
+  RNFastImage
+  React-Codegen
+  React-bridging
+].freeze
+
+def gutenberg_pod(config: GUTENBERG_CONFIG)
+  options = config
```

Comment:
If we don't need to modify the `GUTENBERG_CONFIG` constant, I think we can make a clone of `config` here, like `options = config.clone`? Alternatively, we can remove the constant and change this function to `gutenberg_pod(tag:, commit:, local:)`?